packages/angular/package.json

```diff
@@ -42,7 +43,8 @@
 	],
 	"dependencies": {
 		"@carbon/charts": "^0.58.1",
-		"@carbon/telemetry": "0.1.0"
```

Comment:
`@carbon/telemetry` needs uninstalled, now that it's no longer used. This needs done in all packages in this branch.